docs/virtual_machines/guest_agent_information.md

```diff
@@ -13,11 +13,11 @@ The condition tells that the GA is connected and can be used.
 GA condition on VirtualMachineInstance
 
     status:
-        conditions:
-            - lastProbeTime: "2020-02-28T10:22:59Z"
-            lastTransitionTime: null
-            status: "True"
-            type: AgentConnected
+      conditions:
+      - lastProbeTime: "2020-02-28T10:22:59Z"
+        lastTransitionTime: null
+        status: "True"
+        type: AgentConnected
```

Comment:
This seems an unrelated change.
If you want to reformat things, maybe create a pre-commit for it.